torch/_ops.py

```diff
@@ -470,6 +470,18 @@ def wrapper():
 
         return wrapper()
 
+    # NOTE [HigherOrderOp Schema]
+    # Each invocation of a HigherOrderOperator should have its own schema because
+    # the subgraphs and the arguments can be different even for the same hop.
```

Comment:
Write somewhere that if this is not implemented the we expect your HOP to be functional (e.g. no mutation of inputs, view aliasing has constraints, etc)